CHANGELOG.md

```diff
@@ -13,17 +13,18 @@ Recommendation: for ease of reading, use the following order:
 
 ## [Unreleased]
 ### Changed
-- Discontinued binary releases for MacOS Intel architecture (see #1323) and Windows (as we only ever supported WSL2)
+- Discontinued binary releases for macOS Intel architecture (see #1323) and Windows (as we only ever supported WSL2)
 - Update `merge` crate version `0.1` -> `0.2`
 - Removed `extra.graphql.enableArchetypeInference` config option that was added for compatibility during data migrations
+- (#1402) GQL: Apollo Tracing removed from response. To bring it back, add `x-trace-graphql=1` to request headers.
```

Comment:
My stylistic preference is to put PR links after description, as most people don't need them.